src/jenkins.py

```diff
@@ -249,6 +250,36 @@ def _install_configs(container: ops.Container) -> None:
         container.push(JCASC_CONFIG_FILE_PATH, jenkins_casc_config_file, user=USER, group=GROUP)
 
 
+def _get_groovy_proxy_args(proxy_config: state.ProxyConfig) -> typing.Iterable[str]:
+    """Get proxy arguments for proxy configuration Groovy script.
+
+    Args:
+        proxy_config: The proxy settings to apply.
+
+    Yields:
+        Groovy script proxy arguments.
+    """
+    if proxy_config.https_proxy:
+        yield from (
```

Comment:
Perhaps just yield individual lines? That way Python doesn't need to create a tuple and then yield it out

Reply:
Sure! Thanks for the comment!